flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "Flake for Holochain testing";
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-25.05";
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
```

Comment:
The version of Telegraf available in 25.05 is 1.34 and doesn't support adding default tags, only fields. Don't know what else I've imported at newer versions here, just wanted a newer Telegraf binary at 1.35 or later.